server/db/add_buttons_to_client.sh

```diff
@@ -47,8 +47,9 @@ else
             echo "  Display Name: $unit"
```

Comment:
At line 44, you need to change

```
    while IFS=",", read -r unit phone_number button_serial_number; do
```

To

```
    while IFS=",", read -r unit phone_number button_serial_number locationid; do
```

For the locationid variable to be populated

Reply:
Done!